governance/polls/SparkLend Gnosis - Update sxDAI onboard parameters  - April 15, 2024.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SparkLend Gnosis - Update sxDAI onboard parameter - April 15, 2024
```

Comment:
1. File name does not match poll title
2. Use capitalized words in titles
3. We think the following wording may be better

```suggestion
title: SparkLend Gnosis - Update sxDAI Oracle Onboarding Parameter - April 15, 2024
```